kitchensink/kitchensink.css

```diff
@@ -89,35 +89,3 @@
     visibility: visible;
   }
 }
-
-
```

Comment:
This does not seem related to the IE fix

Reply:
left this as is, since just removed commented out code.